adyenofficial.php

```diff
@@ -92,6 +92,30 @@ class AdyenOfficial extends PaymentModule
      */
     private $adyenPaymentResponseModel;
 
+    /**
+     * Links to show on the Adyen payment method header
+     *
+     * @var array
+     */
+    private static $links = array(
+        array(
+            "label" => "Docs",
+            "url" => "https://docs.adyen.com/plugins/prestashop"
+        ),
+        array(
+            "label" => "Support",
+            "url" => "https://support.adyen.com/hc/en-us/requests/new?ticket_form_id=78764"
+        ),
+        array(
+            "label" => "GitHub",
+            "url" => "https://github.com/Adyen/adyen-prestashop/releases"
+        ),
+        array(
+            "label" => "PrestaShop Marketplace",
+            "url" => "https://addons.prestashop.com"
```

Comment:
shall we redirect to our marketplace page ?

Reply:
At the moment we don't have a marketplace page, will set a reminder to update this when we do